src/WebApp/flask/templates/aztkIns.html

```diff
@@ -58,19 +59,34 @@ <h1 class="mt-5">Cluster Deletion in progress</h1>
         </div>
     </div>
     {% endif %}
-    {% if asset.Status == "Provisioned" %}
+    {% if clusterDetails.Status == "Provisioned" %}
     <div>
         <p> <b>Cluster Information: </b></p>
-        <p> <i>Master Node IP Address : {{asset.Master_Ip_Address}}</i> </p>   
-        <p> <i>Master Node Port Number : {{asset.Master_Port}}</i> </p>  
+        <p> <i>Master Node IP Address : {{clusterDetails.Master_Ip_Address}}</i> </p>   
+        <p> <i>Master Node Port Number : {{clusterDetails.Master_Port}}</i> </p>  
+        <p> <i>Connect via ssh : <b><code>{{clusterDetails.UserName}}@{{clusterDetails.Master_Ip_Address}} -p {{clusterDetails.Master_Port}} -L 127.0.0.1:8888<code></b></i> </p>
```

Comment:
not sure it's a good idea to combine `<b>` and `<i>`. can't see how it looks, but my feeling is that it would looker better as
`<i>Connect via ssh</i>: <code>{{clusterDetails.UserName}}@{{clusterDetails.Master_Ip_Address}} -p {{clusterDetails.Master_Port}} -L 127.0.0.1:8888<code> `
If better looks are confirmed, please use the same pattern for all colon-separated entries on the page.